upp/classes/components.py

```diff
@@ -18,14 +18,15 @@ class Component:
     dirname: Path
     num_jets: int
     num_jets_estimate: int
+    equal_jets: bool
```

Comment:
Can you also add a default value here so that things are consistent with the previous behaviour, thanks

Reply:
Just not to re-do this multiple times I wanted to quickly double check: I think a better place would be changing not the default value for the class but have a behaviour for when `equal_jets` is not in the config (see my other comment below)

Reply:
I'm fine with either approach, you could also implement them both!

Reply:
Added both